.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.6.5
+2.7.2
```

Comment:
is this needed because we have `.tool-versions` and we are using asdf on registers

Reply:
Probably not. I figured it was painless enough to keep around in case people are using rbenv.